### PR TITLE
correct sr-iov terms

### PR DESCRIPTION
**Pull request by Jeanine-tw**

### CODE REVIEW

**Comment by windsonsea** [comment]
This is fixed in #1860

File: docs/zh/docs/network/modules/multus-underlay/install.md [hunk 1]
```diff
@@ -78,23 +78,23 @@
...
-    - `Vlan ID`：可选，SRIOV PF 的 Vlan tag。
-    - `SRIOV Device Plugin Configuration`：用于发现主机上的 SRIOV PF 和 VF device，筛选方式可以为：`vendors`、`devices`、`drivers`、`pfNames`。
+    - `Install SR-IOV CNI`：是否安装 SR-IOV，默认不安装。
+    - `SR-IOV Type`：安装 SR-IOV 的 Multus CRD 实例的类型，有以下几种：
+      - `sriov-overlay`：此类型下，SR-IOV 会与默认 CNI 搭配使用（比如 Calico），这样会在 Pod 中插入两张网卡。
+          分别是默认 CNI 和 SR-IOV 的网卡，前者用于解决 Pod 与集群东西向通信问题；后者用于 Pod 集群南北向通信。
+      - `sriov-standalone`：此类型下，Pod 中只会插入一张 SR-IOV 的网卡，只由其完成与集群东西向和南北向的通信问题。
+    - `SR-IOV CR Name`：Multus CRD 实例的名称。
+    - `Vlan ID`：可选，SR-IOV PF 的 Vlan tag。
+    - `SR-IOV Device Plugin Configuration`：用于发现主机上的 SR-IOV PF 和 VF device，筛选方式可以为：`vendors`、`devices`、`drivers`、`pfNames`。
```
Comment (windsonsea) [inline docs/zh/docs/network/modules/multus-underlay/install.md:94]: UI has the highest priority, which means we need to follow the spelling in UI.

Or, we need to modify the UI firstly and change the screenshot if the correct spelling is SR-IOV.

Comment (Jeanine-tw) [reply docs/zh/docs/network/modules/multus-underlay/install.md]: > UI has the highest priority, which means we need to follow the spelling in UI.
> 
> Or, we need to modify the UI firstly and change the screenshot here if the correct spelling is SR-IOV.

👌，等ui改好再更新文档

**Review by windsonsea — APPROVED** [review]

File: docs/zh/docs/network/modules/multus-underlay/sriov.md [hunk 2]
```diff
@@ -117,7 +117,7 @@ Allocatable:
   ephemeral-storage:             881675818368
   hugepages-1Gi:                 0
   hugepages-2Mi:                 0
-  intel.com/sriov-netdevice:     8      # 此行表示 SR-IOV CNI 成功的发现了该主机上的 VFs 
+  intel.com/-netdevice:     8      # 此行表示 SR-IOV CNI 成功的发现了该主机上的 VFs
```
Comment (windsonsea) [inline docs/zh/docs/network/modules/multus-underlay/sriov.md:120]:
```suggestion
  intel.com/sriov-netdevice:     8      # 此行表示 SR-IOV CNI 成功的发现了该主机上的 VFs 
```
这一行误删了吧

**Pull request merged**